src/components/Header/Header.module.scss

```diff
@@ -0,0 +1,19 @@
+@import url('https://fonts.googleapis.com/css2?family=KoHo&display=swap');
+
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 0 1rem;
+  height: 5rem;
```

Comment:
I have a comment above about pixels vs rems. Let's use REMs especially for fonts. The rest is up to your preferences. In the moment you understand the difference between it, feel free to use anything.

But! the important part. Let's stay consistent. So for the not-font properties:

![CleanShot 2023-10-06 at 15 12 04](https://github.com/KamilaSemotanova/tracker/assets/12488648/73066429-1873-4b69-8b58-efd5b52783ab)
![CleanShot 2023-10-06 at 15 12 19](https://github.com/KamilaSemotanova/tracker/assets/12488648/c336ee7d-fae7-4d40-b835-b10a4d573013)

either rems or pixels.